enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/BoltInteraction.java

```diff
@@ -68,9 +69,10 @@
 
     EnterpriseAuthManager authManager;
 
-    public BoltInteraction( Neo4jWithSocket server ) throws IOException
+    public BoltInteraction( Neo4jWithSocket server, Consumer<Map<Setting<?>, String>> configure ) throws IOException
     {
         this.server = server;
+        server.restartDatabase( configure );
```

Comment:
Maybe you can create the Neo4jWithSocket in here and just pass configure instead